tests/new_unit_tests/Commands/Connection/SetCommandTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+use Pantheon\Terminus\Commands\Connection\SetCommand;
+use Prophecy\Prophet;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Connection\SetCommand
+ */
+class SetCommandTest extends ConnectionCommandTest
+{
+    /**
+     * Test suite setup
+     *
+     * @return void
+     */
+    protected function setup()
+    {
+        parent::setUp();
+
+        $this->command = new SetCommand($this->getConfig());
+        $this->command->setLogger($this->logger);
+    }
+
+    /**
+     * Exercises connection:set git mode
+     *
+     * @return void
+     *
+     * @vcr site_set-connection-mode_git
+     */
+    public function testConnectionSetGit()
```

Comment:
This unit test is identical to the Behat one:

```
+  @vcr site_set-connection-mode_git
 +  Scenario: Setting connection mode to git
 +    When I run "terminus connection:set [[test_site_name]].dev git"
 +    Then I should see a notice message: Enabling git push mode for "dev"
```

Is there any point in doing these if they're exactly the same?